app/models/evidence.rb

```diff
@@ -20,6 +20,8 @@ class Evidence < ApplicationRecord
   validates :issue, presence: true, associated: true
   validates :node, presence: true, associated: true
 
+  validate :issue_cannot_be_on_another_project
```

Comment:
I don't love the naming. How about something more positive

`validate :is_scoped_to_project`